packages/create-webpack-app/src/plopfile.ts

```diff
@@ -0,0 +1,179 @@
+import { NodePlopAPI } from "./types";
+import { resolve } from "path";
+import ejs from "ejs";
+/* eslint-disable no-unused-vars */
```

Comment:
Let's not disable the rule for the whole file, prefer to use `//eslint-disable-next-line`

Reply:
changed this in latest commits